### PR TITLE
OS-738 style fixes

### PR DESCRIPTION
**Pull request by retif**
Fixes for branch info block, requested in previous review.

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: modules/openy_features/openy_block/modules/openy_block_branch_contacts_info/templates/block-branch-contacts-info.html.twig [hunk 1]
```diff
@@ -43,6 +44,16 @@
           </a>
         </div>
       </div>
+      {% if fax is not empty %}
+        <div class="wrapper-field-location-fax">
+          <div class="field-location-fax">
+            <a href="tel:{{ fax }}">
```
Comment (anpolimus) [inline modules/openy_features/openy_block/modules/openy_block_branch_contacts_info/templates/block-branch-contacts-info.html.twig]:
```suggestion
            <a href="{{ 'tel:'|t }}{{ fax }}">
```

**Review by anpolimus — APPROVED** [review]

**Comment by anpolimus** [comment]
@podarok this PR has fixed for items, requested by @ddrozdik here: https://github.com/ymcatwincities/openy/pull/1936

**Review by podarok — APPROVED** [review]

File: modules/openy_features/openy_block/modules/openy_block_branch_contacts_info/templates/block-branch-contacts-info.html.twig [hunk 2]
```diff
@@ -4,9 +4,10 @@
  * Default theme implementation to display branch contacts info.
  *
  * Available variables:
+ *  - node_bundle: will return node bundle
```
Comment (ddrozdik) [inline modules/openy_features/openy_block/modules/openy_block_branch_contacts_info/templates/block-branch-contacts-info.html.twig:7]: please remove extra space before the dash

**Review by ddrozdik — APPROVED** [review]

**Pull request merged**